web/package/d-installer-web.spec

```diff
@@ -0,0 +1,59 @@
+#
+# spec file for package cockpit-machines
+#
+# Copyright (c) 2022 SUSE LLC
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+
+
+Name:           d-installer-web
+Version:        0
+Release:        0
+Summary:        Web-based user interface for D-Installer
+License:        GPL-2.0-only
+URL:            https://github.com/yast/d-installer
+# source_validator insists that if obscpio has no version then
+# tarball must neither
+Source:         d-installer-web.tar
```

Comment:
why not compressed?

Reply:
so maybe we should document it, as it is different then what we have in yast, so it can confuse people.